docs/README.md

```diff
@@ -175,23 +175,11 @@ See [Third-Party Commitizen Templates](third-party-commitizen.md).
 
 ## FAQ
 
-### Why are `revert` and `chore` valid types in the check pattern of cz conventional_commits but not types we can select?
-
-`revert` and `chore` are added to the "pattern" in `cz check` in order to prevent backward errors, but officially they are not part of conventional commits, we are using the latest [types from Angular](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type) (they used to but were removed).
-However, you can create a customized `cz` with those extra types. (See [Customization](https://commitizen-tools.github.io/commitizen/customization))
-
-See more discussion in issue [#142](https://github.com/commitizen-tools/commitizen/issues/142) and [#36](https://github.com/commitizen-tools/commitizen/issues/36)
-
-### How to handle revert commits?
-
-```sh
-git revert --no-commit <SHA>
-git commit -m "revert: foo bar"
-```
+See [FAQ - commitizen](https://commitizen-tools.github.io/commitizen/faq/).
```

Comment:
@Lee-W Should I replace the URL with filename? It seems that filenames will be rendered as URL in generated documentation.
For example, `contributing.md` will turn into https://commitizen-tools.github.io/commitizen/contributing/

Reply:
Yep, that would be even much better. 💯 didn't notice this one.

Reply:
Should I also replace those URLs inside other files? But I am not sure how to reference files not in the same directory.  
For example, near the end of this line, `docs/tutorials/gitlab_ci.md` refers to `docs/bump.md`.

https://github.com/commitizen-tools/commitizen/blob/061f47c1434f2df2da49579ff92e489001ecd3ec/docs/tutorials/gitlab_ci.md#L13

There seems to exist 2 valid path? `../bump.md` and `/docs/bump.md`, I am not sure which one is better and whether they will work in the final documentation.

Reply:
I would prefer `../bump.md` in this case. If my understanding is correct, when deployed to gh pages `/docs/bump.md` becomes `https://commitizen-tools.github.io/docs/bump.md`

Reply:
And yes, if you found any other problem, it would be better if you could fix them as well

Reply:
I have updated all the links I can find in documentation, and I found that absolute path like `/docs/bump.md` is not support by MkDocs, the documentation is [here](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages).